config/management/src/validator_operator.rs

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    constants,
+    error::Error,
+    secure_backend::StorageLocation::{LocalStorage, RemoteStorage},
+    SecureBackends,
+};
+use libra_global_constants::{OPERATOR_KEY, OWNER_ACCOUNT, OWNER_KEY};
+use libra_secure_storage::{CryptoStorage, KVStorage, Value};
+use libra_secure_time::{RealTimeService, TimeService};
+use libra_types::{
+    account_address,
+    account_address::AccountAddress,
+    transaction::{RawTransaction, Script, SignedTransaction, Transaction},
+};
+use std::{str::FromStr, time::Duration};
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+pub struct ValidatorOperator {
+    #[structopt(long)]
+    operator_name: String,
+    #[structopt(flatten)]
+    backends: SecureBackends,
+}
+
+impl ValidatorOperator {
+    pub fn execute(self) -> Result<Transaction, Error> {
+        // Fetch the operator key from remote storage using the operator_name and derive an address
+        let operator_account = self.fetch_operator_account()?;
+
+        // Create the transaction script that sets the validator operator for the owner
+        let set_operator_script =
+            transaction_builder::encode_set_validator_operator_script(operator_account);
+
+        // Create and sign the set-operator transaction
+        let set_operator_tx = self.create_set_operator_transaction(set_operator_script)?;
+
+        // Upload the set-operator transaction to shared storage
+        match self.backends.remote {
+            None => return Err(Error::RemoteStorageMissing),
+            Some(remote_config) => {
+                let mut remote_storage = remote_config.create_storage(RemoteStorage)?;
+                remote_storage
+                    .set(
+                        constants::VALIDATOR_OPERATOR,
+                        Value::Transaction(set_operator_tx.clone()),
+                    )
+                    .map_err(|e| {
+                        Error::RemoteStorageWriteError(constants::VALIDATOR_OPERATOR, e.to_string())
+                    })?;
+            }
+        };
+
+        Ok(set_operator_tx)
+    }
+
+    /// Creates and returns a signed set-operator transaction to be uploaded to the remote storage.
+    fn create_set_operator_transaction(&self, script: Script) -> Result<Transaction, Error> {
```

Comment:
due to "reasons", let's avoid making this a transaction and instead just upload the name of the operator. If you want, we can put this under a test flag or comment it out... because I can imagine post genesis, this will be more of a utility app.  or come up with some alternative...

Reply:
Done, thanks! For now, we just upload the name, and then later use that name to set the operator account in genesis.